web/download_manager.js

```diff
@@ -86,20 +86,20 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC || CHROME')) {
     },
 
     download: function DownloadManager_download(blob, url, filename) {
-      if (!URL) {
```

Comment:
I'm not sure that I follow, but why isn't it sufficient to *only* change this line to e.g. `if (!URL || pdfjsLib.PDFJS.disableCreateObjectURL) {` instead of making all the other changes here!?